### DISCUSSION TITLE
duplicate recordings

### DISCUSSION THREAD

**Discussion by Cah1982** [blakeblackshear/frigate#7727] [General]
just curious to how this happens, I know recording some time overlap near start / end but on the bellow, i dont know why

![image](https://github.com/blakeblackshear/frigate/assets/44786907/93eff049-c967-4979-a186-6261559eddef)

**Comment by NickM-27** [comment]
you are going to need to provide more info. Was there actually 2 separate people, what is your expected behavior, etc.

**Comment by Cah1982** [comment]
no just me in both recordings

the longest recording is correct, as it records after i left the room, bottom stops with me still in front of camera

**Reply by NickM-27** [reply]
In that case 2 different people were detected despite there only being 1

**Comment by dopeytree** [comment]
I have noticed this aswell. I was going to fiddle with the setting for how many frames are considered for a detection.

**Comment by ghost** [comment]
Good morning,
I noticed the same kind of thing, when there were several people (3 here) in the same room (here the Living Room)
https://i.goopics.net/9pjl05.jpg

I don't know if this is the expected operation, but I understand that several people in the same room "can" trigger several detections (with different IDs) and therefore several events, therefore several clips etc...

My config: https://pastebin.com/A5cC1085

Is it possible, through config, to avoid this multitude (sometimes several dozen in a few minutes) of events/clips/mqtt messages?

From my point of view, I need to know that there is movement at a moment in a room, but there is no need to have "all" of the movements/detections/clips.
I plan to be "notified" by SMS, and I am afraid in this mode of operation, of being inundated with SMS in a few minutes :)

It's possible ?

**Reply by NickM-27** [reply]
events are synonymous with tracked objects, so if there are multiple people then multiple events is expected. Frigate has existing logic to cleanup these events every 5 minutes so that helps, but if they are different lengths then they are treated as unique.

**Reply by ghost** [reply]
ok, thank you very much for the explanation, I understand better now with this concept.
